guides/InstallationAdminGuide.md

```diff
@@ -361,6 +363,8 @@ Server administrators are able to give institutions an arbitrary base URL. This
 The arbitrary base URL can be entered in the Institution URL edit box. The Institution URL should be fully qualified. It is not possible to overwrite the other institution’s URL space, for example: ‘http://equella.myinstitution:4012/doco/qa2/’ will conflict with  ‘http://equella.myinstitution:4012/doco/’. This will be disallowed and will result in the following message:
 **‘URL must not 'overwrite' an existing institution's URL space, in this case http://equella.myinstitution:4012/doco/qa2/. This may cause this institution to work incorrectly’.**
 
+Note: The first two examples above conflict with the default setting for `admin.url`. If using the top level for your repository change `admin.url` in `learningedge-config/mandatory-config.properties` to include `/adm` or similar.
+
```

Comment:
I would change this to:

Note: The `admin.url` needs to have it's own URL space.  While possible to have your institution as the top level (my.oeq.org) and the `admin.url` be a context of it (my.oeq.org/adm/), it's not recommended.  A _cleaner_ approach would be `admin.my.oeq.org`.  A _more secure_ approach would be to set the `admin.url` to an IP address of the given server so it's only accessible behind your organization's firewall.

Reply:
@goetzk - I would back off the use of  'recommended' for 
> Server administrators are able to give institutions an arbitrary base URL but it is **recommended** that they contain a context (`/something/` after the address)

As long as the system administrator is aware of the implications, a context is not needed for institutions, and multiple adopters use this flow.

Otherwise, looks good!

Reply:
Sure, an issue ticket for /Equella works.  
> On a related topic, should I open an issue against equella asking that admin be protected from institutions masking it just as other institutions are?

Reply:
@goetzk - I would remove 'recommended'.